README.md

```diff
@@ -1 +1,54 @@
 # Shorebird (un)installer
+
+**❗️ Note: This project is under heavy development. Things will change frequently and none of the code is ready for production use. We will do our best to keep the documentation up-to-date.**
+
+## Getting Started 🚀
+
+1. 🔑 Request an API Key
+1. 🐦 Install Shorebird
+
+   ```
+   dart pub global activate --source git https://github.com/shorebirdtech/shorebird --git-path packages/shorebird_cli
```

Comment:
This is temporary until we have a proper `install.sh` and assumes users have Dart installed.

Reply:
lgtm